### PR TITLE
Review by AJ

### PR DESCRIPTION
**Pull request by andreas-junghanns**
Mostly language improvements, some non-normative text turned to normative, one limitation deleted (multiple reasons).

### CODE REVIEW

File: docs/index.adoc [hunk 1]
```diff
@@ -432,15 +430,13 @@ Therefore, this constellation is not applicable to this layered standard and the
 
 Limitations for binary FMUs:
 
-* Normally only variables in the memory segments owned by the main FMU binary `{.dll, .so}` are accessible by the XCP service, i.e. variables defined in source code modules and in statically linked libraries.
+* Normally only variables in the memory segments owned by the main FMU binary `{.dll, .so}` are accessible by the XCP service, i.e., variables defined in source code modules and in statically linked libraries.
 Accessing variables in the memory of indirectly loaded dynamic libraries requires a special implementation which is out-of-scope for this layered standard.
 
-Limitations for variant <<External XCP Service,External XCP Service>>:
```
Comment (PTaeuberDS) [inline docs/index.adoc:438]: We had in mind that there could be a future version of this layered standard that defines a way for the FMU to announce memory segments to the importer. But I'm also fine with removing this limitation.

File: docs/index.adoc [hunk 2]
```diff
@@ -339,29 +337,32 @@ If the value of structural parameter `org.fmi-standard.fmi-ls-xcp.EnableInternal
...
 
 If `Configuration Mode` was not entered and the value of structural parameter `org.fmi-standard.fmi-ls-xcp.EnableInternalXcpService` is `true`, the FMU must start the XCP service in `fmi3EnterInitializationMode` at the latest.
 In this case, it is not possible to perform calibration before `Initialization Mode` is entered or to configure the XCP connection settings.
 
-_[If the importer does not support `Configuration Mode` it is not able to control whether or not the internal XCP service shall be started._
-_In this case the default value of the `start` attribute of variable `org.fmi-standard.fmi-ls-xcp.EnableInternalXcpService` determines if the XCP service is started.]_
+If the importer does not support `Configuration Mode`, it is not able to control whether or not the internal XCP service shall be started.
+In this case the default value of the `start` attribute of variable `org.fmi-standard.fmi-ls-xcp.EnableInternalXcpService` determines if the XCP service is started.
+
+_[Note that it is not possible to manually change the `start` value for `org.fmi-standard.fmi-ls-xcp.EnableInternalXcpService` to change the behavior of the internal XCP service._
```
[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/index.adoc:348]: I clarified this in the PR. I think it helps (as a non-normative comment only) to remind people that what is in the modelDescription.xml is only the default implemented inside the FMU.

File: docs/index.adoc [hunk 3]
```diff
@@ -389,19 +390,18 @@ image::images/XCP-Communication-external-Slave.svg[width=80%, align="center"]
...
+When an external XCP service is used, the importer has to make XCP service calls and is responsible for creating events for its XCP service.
+However, in contrast to the FMU with an internal XCP support, the importer does not have the knowledge about the inner structure of the virtual ECU and therefore it is not able to create specific XCP events, e.g., one event for each task.
 
-_[The importer is responsible for matching the `IF_DATA XCP` section in the A2L file to the implementation of the XCP service._
-_If necessary, the `IF_DATA XCP` section must be added or replaced in the A2L file delivered with the FMU.]_
+This means, by default, the importer can only make XCP calls at certain points in time, e.g., at communication points in Co-Simulation, ignoring specific task-related XCP events.
+The importer is responsible for matching the `IF_DATA XCP` section in the A2L file to the implementation of the XCP service.
+If necessary, the `IF_DATA XCP` section must be added or replaced in the A2L file delivered with the FMU, before making it available ot the MCD tool.
```
[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/index.adoc:398]: Fixed.

File: docs/index.adoc [hunk 4]
```diff
@@ -432,15 +430,13 @@ Therefore, this constellation is not applicable to this layered standard and the
...
-
-* This layered standard does not describe how the importer can determine the different memory segments within the built FMU.
-Therefore, page handling is not supported in the case of an external XCP service.
-
 [bibliography]
 == References
 
 - [[[PW13]]] Preston-Werner, T. (2013): **Semantic Versioning 2.0.0**.  https://semver.org/spec/v2.0.0.html
+- [[[ASAP2]]] ASAM MCD-2 MC. https://www.asam.net/standards/detail/mcd-2-mc/
+- [[[RFC2119]]] RFC 2119. https://tools.ietf.org/html/rfc2119
```
[the remark this answers is not in the document]
Comment (andreas-junghanns) [reply docs/index.adoc:441]: Yes, I wasn´t too sure how to fix the original references. But I think it looks more impressive if we have at least a few bib entries. Only one? Then I would opt to remove that and make that reference also just inline. I am ok either way (preferring slightly the current proposal).

File: docs/index.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/index.adoc:438]: This is NOT the job of this standard. Memory segments are defined inside the a2l file.

Comment (PTaeuberDS) [reply docs/index.adoc:438]: Yes, you are right. This was more related to source code FMUs, where the memory layout is not known yet. The FMU could announce which memory segments are available and the importer could enhance the A2L file with the MOD_PAR info.
So, the limitation would apply to source code FMUs with external XCP service. But we can also remove it completely.

**Review by PTaeuberDS — APPROVED** [review]

**Pull request merged**